src/Tokenizer/Token.php

```diff
@@ -300,6 +300,8 @@ public static function getNameForId(int $id): ?string
 
         $name = token_name($id);
 
+        \assert('' !== $name);
```

Comment:
can `token_name` ever return empty string ?

Reply:
maybe let's convert to phpstan exception and wait remove exception when https://github.com/phpstan/phpstan/issues/11808 is solved

Reply:
https://github.com/PHP-CS-Fixer/PHP-CS-Fixer/pull/8245